<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Choose</class>
 <widget class="QDialog" name="Choose">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>计算器</string>
  </property>
  <widget class="QPushButton" name="Calculate1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>傅里叶空间
像素大小</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Calculate2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>衍射束斑半径</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Calculate3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>剂量</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Calculate4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Calculate5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Calculate6">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
